data/hai/unfettered 1 alphas.txt

```diff
@@ -0,0 +1,1532 @@
+			choice
+				`	"There are logs on my ship, apparently speaking about the source of the jump drives. I will send them to you."`
+					goto showlogs
+				`	"I understand, but it will take a while to earn my trust back."`
+					decline
+				`	"I don't want to have anything to do with you Unfettered anymore."`
+			`	"It did not have to be this way, but if you want to go, we will not detain you. Do not expect any shares of the loot from us though."`
+			action
+				clear "salary: Unfettered"
+				fail "<uhai> 9: Capture The Alpha Auxiliary"
+				set "unfettered: betrayed"
+
+event "unfettered: korath back up"
+
+
+
+mission "<uhai> 9: Alpha Base"
+	landing
+	name "Time for questions"
+	description "You have learned some surprising things about the Unfettered's possible role with the Alphas. You should ask Mera for clarifications since she directed you here."
+	source "Avalon"
+	destination "Darkcloak"
+	to offer
+		has "<uhai> 9: Capture The Alpha Auxiliary: active"
+		has "unfettered: captured the auxiliary"
+		has "ship model: Auxiliary"
+	on offer
+		clear "unfettered: captured the auxiliary"
+		set "alpha: host found"
+		conversation
+			`You do a detailed scan of the Alpha base to see if there are any more jump drives, or anyone who could have witnessed the fight. You cannot retrieve any information: it seems to either be too deep or shielded from scans in some way. You will have to go in blind.`
+			`	Surprisingly, the doors of the hangar open to let you in. Maybe it is an automated response to this ship, or whoever is down here assumed the Alphas won the fight. That said, if there is anyone here, it's probably a skeleton crew.`
+			`	You and your crew carefully exit your ship, but you see no signs of life here. They could be hidden, so you proceed cautiously, looking for the jump drive vault. On the way you hear sounds - sounds that definitely did not originate from humans. Heading that way, you find a brig full of prisoners, with a lizard head displayed above the cells. They seem to have been mistreated by the Alphas, and you find what appears to be a crudely rigged translation device next to their cell, with information regarding these Korath, an alien species located north-east of human space that sometimes engages in raiding. They apparently used to control the territories nearby that are now owned by automata.`
+			`	The captives themselves must have endured a lot, you should take them to Darkcloak and ask the Chiefs for help. You notice one of them has a very serious injury to their leg, it seems that it has been partially replaced by a crude prosthetic. He may not survive the transport.`
```

Comment:
```suggestion
			`	The captives themselves must have endured a lot, so you should take them to Darkcloak and ask the Chiefs for help. You notice one of them has a very serious injury to his leg, and it seems that it has been partially replaced by a crude prosthetic. He may not survive the transport.`
```

Reply:
We’ll later it says He.

Reply:
~~right~~ well congratulations, its a boy